store/presenters/presenters.go

```diff
@@ -485,3 +485,85 @@ func (a NewAccount) GetID() string {
 func (a NewAccount) GetName() string {
 	return "keys"
 }
+
+// Tx is a jsonapi wrapper for an Ethereum Transaction.
+type Tx struct {
+	*models.Tx
+}
+
+// GetID returns the jsonapi ID.
+func (t Tx) GetID() string {
+	return t.Hash.String()
+}
+
+// GetName returns the collection name for jsonapi.
+func (Tx) GetName() string {
+	return "transactions"
+}
+
+type txJSON struct {
+	Confirmed bool   `json:"confirmed"`
+	Data      string `json:"data"`
+	From      string `json:"from"`
+	GasLimit  uint64 `json:"gasLimit"`
+	GasPrice  string `json:"gasPrice"`
+	Hash      string `json:"hash"`
+	Hex       string `json:"rawHex"`
+	Nonce     uint64 `json:"nonce"`
+	SentAt    uint64 `json:"sentAt"`
+	To        string `json:"to"`
+	Value     string `json:"value"`
+}
+
+// MarshalJSON returns the Transaction as json.
+func (t Tx) MarshalJSON() ([]byte, error) {
+	txj := txJSON{
+		Confirmed: t.Confirmed,
+		Data:      common.Bytes2Hex(t.Data),
+		From:      t.From.Hex(),
+		GasLimit:  t.GasLimit,
+		GasPrice:  t.GasPrice.String(),
+		Hash:      t.Hash.Hex(),
+		Hex:       t.Hex,
+		Nonce:     t.Nonce,
+		SentAt:    t.SentAt,
+		To:        t.To.Hex(),
+		Value:     t.Value.String(),
+	}
+	return json.Marshal(&txj)
+}
+
+// UnmarshalJSON deserializes the Transaction from json.
+func (t *Tx) UnmarshalJSON(b []byte) error {
+	var txj txJSON
+	if err := json.Unmarshal(b, &txj); err != nil {
+		return err
+	}
+
+	gasPrice, ok := big.NewInt(0).SetString(txj.GasPrice, 10)
+	if !ok {
+		return fmt.Errorf("presents.Tx could not unmarshal gasPrice from JSON: %v", b)
```

Comment:
It's interesting you revert back to primitive types like `string` and `uint64`. We have helper types like `models.Big` and `hexutil.Big` to assist with all this unmarshaling.